dir.traversal.targets

```diff
@@ -1,7 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
-  <Import Project="$(MSBuildToolsPath)\Microsoft.Common.targets" />
-
```

Comment:
I kind of like this approach as well, I don't actually remember why I ever did that import :) However I was just pointing out that it was different and thus inconsistent in the CoreFX repo, we could of course do the same fix over there.